### ISSUE TITLE
No data past 8/1 on nugettrends.com

### ISSUE THREAD

**Issue by sunandabalu** [dotnet/nuget-trends#165]
Looks like the data flow stopped after 8/1 to nugettrends.com 
 https://nugettrends.com/packages?months=3&ids=upgrade-assistant
https://www.nuget.org/packages/upgrade-assistant/

![image](https://user-images.githubusercontent.com/16616736/129634422-4920f41d-e9a4-452d-b496-8668e32c48bc.png)

**Comment by joaopgrassi** [comment]
Hey,
Thanks for reaching out about this issue. I did some initial investigations and there was indeed a gap while the daily download workers wasn't doing its job. Did some container troubleshooting and triggered the jobs again and now it seems to be back at it.

![image](https://user-images.githubusercontent.com/5938087/129809299-5cd352be-1f4b-45b8-93f5-6c731bb50056.png)


Some context for history: We recently did a revamp in our deployment strategy and somehow something got broken in between.  

Will keep looking to see what's happening and if it's stable. Will also try to see what we can do to find out about these type of issues automatically and get alerts instead of GitHub issues :sweat_smile:

**Comment by bruno-garcia** [comment]
I can set up a Sentry alert that kicks off if we don't get transactions for the workers runs

**Comment by joaopgrassi** [comment]
@bruno-garcia that would be good :). Once that is done I think we can close this. I opened another issue to investigate more.. also to try to fix some error that is happening sometimes in the `DailyDownloadPackageIdPublisher`.

**Comment by bruno-garcia** [comment]
Got data after that date, then there was another gap raised on #233 
Data lives on another server so I might move before decomissioning that
